routes/adminRoutes.js

```diff
@@ -59,8 +58,34 @@ router.delete('/team/:teamId', async (req, res) => {
   }
 });
 
-router.delete('/curler/:curlerId', async (req, res) => {
+router.put('/team/:teamId', async (req, res) => {
+  try {
+    const id = req.params.teamId;
+    const { name, affiliation, note } = req.body;
```

Comment:
sorry, just found a bug. Please use identical field names from the api endpoints doc and not from the table. They may not necessarily be the same. orgId instead of affiliation